src/index.vue

```diff
@@ -8,7 +8,7 @@
       :style="{ backgroundColor }">
       <div class="velmld-spinner">
         <slot name="default">
-          <component :is="spinner" :color="spinnerColor"></component>
+          <component :is="spinner" :color="spinnerColor" :size="spinnerSize" :duration="spinnerDuration"></component>
```

Comment:
Actually, we can use `props` directly, Unnecessary to watch this value.

This is my fault. Can you please fix this line to

```suggestion
          <component :is="spinner" :color="color" :size="`${size}px`" :duration="`${duration}s`"></component>
```